src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      return (this.result += value);
+    },
+
+    subtract(value) {
+      return (this.result -= value);
+    },
+
+    multiply(value) {
+      return (this.result *= value);
+    },
+
+    divide(value) {
+      if (value === 0) {
+        return undefined;
+      }
+
+      return (this.result /= value);
```

Comment:
[CODE STYLE] - The divide method should return the calculator object after performing the operation to maintain method chaining. Additionally, returning undefined when dividing by zero is not consistent with the rest of the API. Consider throwing an error instead.